eng/performance/blazor_scenarios.proj

```diff
@@ -1,11 +1,18 @@
 <Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
 
-  <ItemGroup>
-    <HelixCorrelationPayload Include="$(CorrelationPayloadDirectory)">
-      <PayloadDirectory>%(Identity)</PayloadDirectory>
-    </HelixCorrelationPayload>
-  </ItemGroup>
-  
+  <PropertyGroup>
+    <LogDirectory Condition="'$(TargetsWindows)' == 'true'">%HELIX_WORKITEM_UPLOAD_ROOT%\</LogDirectory>
+    <LogDirectory Condition="'$(TargetsWindows)' != 'true'">%24{HELIX_WORKITEM_UPLOAD_ROOT}/</LogDirectory>
+
+    <PerfLabTargetFrameworksEnvVar Condition="'$(TargetsWindows)' == 'true'">%PERFLAB_TARGET_FRAMEWORKS%</PerfLabTargetFrameworksEnvVar>
+    <PerfLabTargetFrameworksEnvVar Condition="'$(TargetsWindows)' != 'true'">%24{PERFLAB_TARGET_FRAMEWORKS}</PerfLabTargetFrameworksEnvVar>
+    <PublishArgs>--msbuild "/p:_TrimmerDumpDependencies=true" --msbuild /warnaserror:NU1602,NU1604 --msbuild-static AdditionalMonoLinkerOptions=%27&quot;%24(AdditionalMonoLinkerOptions) --dump-dependencies&quot;%27 --binlog $(LogDirectory)blazor_publish.binlog</PublishArgs>
```

Comment:
Passing the same arg to python's argparser multiple times has an effect of ignoring all args but the last one. I doubt `_TrimmerDumpDependencies=true` was set.
E.g. running `python3 ./pre.py publish -f net7.0 --readonly-dotnet --has-workload  --msbuild "/p:HybridGlobalization=true" --msbuild "/p:RunAOTCompilation=true" --msbuild /warnaserror:NU1602,NU1604` results in:
`dotnet publish /workspaces/performance/src/scenarios/blazorpizza/app/BlazingPizza.sln --configuration Release /p:NuGetPackageRoot=/workspaces/performance/artifacts/packages/ /p:RestorePackagesPath=/workspaces/performance/artifacts/packages/ /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:PublishDir=/workspaces/performance/src/scenarios/blazorpizza/pub --framework net7.0 /warnaserror:NU1602,NU1604 -p:WasmNativeWorkload=false /p:EnableWindowsTargeting=true`
and running with e.g. `--msbuild "/p:HybridGlobalization=true" --msbuild "/p:RunAOTCompilation=true" ` results in `/p:RunAOTCompilation=true -p:WasmNativeWorkload=false /p:EnableWindowsTargeting=true`